packages/verification-vue-component/src/components/Verification/items/cards/VerificationCard.vue

```diff
@@ -158,28 +158,31 @@ export default {
         verificationItem: {
             type: Object,
             required: true
+        },
+        net: {
+            type: String,
+            required: true
         }
     },
     computed: {
-        net() {
-            if (this.$parent.$parent.certifactionEthVerifier.certifactionEthClient.eth.currentProvider.host.indexOf('ropsten') >= 0) {
-                return 'ropsten.etherscan.io'
-            } else {
-                return 'etherscan.io'
-            }
+        registerEvents() {
+            return this.verificationItem.events ? this.verificationItem.events.filter(event => ['register', 'certify'].indexOf(event.scope) >= 0) : null
+        },
+        revokeEvents() {
+            return this.verificationItem.events ? this.verificationItem.events.filter(event => event.scope === 'revoke') : null
         },
         verificationItemType() {
-            if (this.verificationItem.error) {
-                return 'technicalProblem'
-            }
-
             if (this.verificationItem.hashed === undefined || this.verificationItem.hashed === false) {
                 return 'ShadowItem'
             }
 
+            if (this.verificationItem.offchainError && Object.keys(this.verificationItem.errors).length > 0) {
```

Comment:
It's just `this.verificationItem.error` and you can probably also just check it's set.

Reply:
Fixed the typo but saw while testing there is an empty error object when all is fine

Reply:
And just `if (this.verificationItem.offchainError && this.verificationItem.error)` doesn't work?

Reply:
Adjusted